src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/MathFunctionsTest.java

```diff
@@ -315,4 +315,46 @@ public void coshWithValueArgument() {
                         "Math.cosh(0)"));
     }
 
+    @Test
+    public void powerWithPropertyArgument() {
+        String query = "SELECT POWER(age, 2) " +
+                "FROM bank";
```

Comment:
np: make it as single string instead of spreading on two lines. More readable. Here and for other test cases added in this class.

Reply:
@abbashus Done. Thanks!